db/migrate/20230711111829_create_comments.rb

```diff
@@ -2,7 +2,7 @@ class CreateComments < ActiveRecord::Migration[7.0]
   def change
     create_table :comments do |t|
       t.integer :author_id, null: false
-      t,integer :post_id, null: false
+      t.integer :post_id, null: false
       t.text :text, null: false
       t.references :user, foreign_key: { on_delete: :cascade }
       t.references :post, foreign_key: { on_delete: :cascade }
```

Comment:
- [ ] I got below error message while trying to run `rails db:migrate` on your app. Kindly check the error message and fix where the issue is.

### Error
![migrate-error](https://github.com/ridabensalem/blog-app/assets/26597432/95ae3b17-dbd0-4db7-a7a7-1597a5a6033e)